frontend_app_thread/test/Thread.spec.js

```diff
@@ -26,10 +25,12 @@ describe('<Thread />', () => {
     t: key => key
   }
 
-  mockGetThreadContent200(debug.config.apiUrl, contentThread.thread.workspace_id, contentThread.thread.content_id, contentThread.thread)
-  mockPutMyselfThreadRead200(debug.config.apiUrl, props.loggedUser.user_id, contentThread.thread.workspace_id, contentThread.thread.content_id)
-  mockGetThreadComment200(debug.config.apiUrl, contentThread.thread.workspace_id, contentThread.thread.content_id, contentThread.commentList)
-  mockGetThreadRevision200(debug.config.apiUrl, contentThread.thread.workspace_id, contentThread.thread.content_id, contentThread.revisionList)
+  const apiUrl = 'http://localhost/api/v2'
```

Comment:
Is there still v2 in that url?

Reply:
my last commit was wrong, I went too fast